From 209c44c883335cb6b579da2dda6a61bf33d5b8c2 Mon Sep 17 00:00:00 2001
From: Du Yilong <duyilong@wxiat.com>
Date: Fri, 21 Oct 2022 15:40:07 +0800
Subject: [PATCH] sw64: kvm: add qemu fw_cfg device to chip_vt.dts
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 209c44c883335cb6b579da2dda6a61bf33d5b8c2
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I645LQ

--------------------------------

Signed-off-by: Du Yilong <duyilong@wxiat.com>
Reviewed-by: He Sheng <hesheng@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/boot/dts/chip_vt.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/sw_64/boot/dts/chip_vt.dts b/arch/sw_64/boot/dts/chip_vt.dts
index abad29dee97e..f26285367f98 100644
--- a/arch/sw_64/boot/dts/chip_vt.dts
+++ b/arch/sw_64/boot/dts/chip_vt.dts
@@ -46,5 +46,10 @@ misc: misc0@8036 {
 			clock-frequency = <24000000>;
 			status = "okay";
 		};
+		fw_cfg: fw_cfg@8049 {
+			dma-coherent;
+			reg = <0x8049 0x20000000 0x0 0x18>;
+			compatible = "qemu,fw-cfg-mmio";
+		};
 	};
 };
-- 
2.33.0

